support-frontend/app/views/router.scala.html

```diff
@@ -23,7 +23,7 @@
 @main(
   title = "Support the Guardian | Checkout",
   mainElement = assets.getSsrCacheContentsAsHtml(divId = "router", file = "ssr-holding-content.html"),
-  mainJsBundle = RefPath("[countryGroupId]/router.js"),
+  mainJsBundle = if (serversideTests.get("lazyLoading").contains(Variant)) RefPath("[countryGroupId]/lazyRouter.js")  else RefPath("[countryGroupId]/router.js"),
```

Comment:
Ah, this is where it makes the decision of what router to take. Should I be able to find these in the network tab under `*.js` bundles?

Reply:
Indeed 👍